docs/blog/2019-09-26-gatsby-perf/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Web Performance 102: Keeping Gatsby Sites Blazing Fast"
+author: Dustin Schau
+date: 2018-09-26
+image: images/rocket.jpg
+showImageInArticle: false
+tags: ["performance", "v2", "lighthouse", "webpagetest"]
+---
+
+We say it all the time: Gatsby sites are blazing fast. 
+
+It’s a great tagline. But for you, that might raise more questions:
+What’s the value in being blazing fast?
+How do you know it’s blazing fast?
+How can it be even more blazing (incineratingly?) fast?
+
+These are questions the Gatsby core team has thought about recently, since our recent release of Gatsby v2 improved upon Gatsby v1's gold standard to squeeze even more performance out of every site. 
+
+And what are some simple, meaningful metrics to _know_ if your site is blazing fast? 
+
+Let's get to it.
+
+> At the BBC, 10% of users are lost for every additional second the site takes to load.
+
+## Measuring Speed
+
+It can oftentimes be overwhelming and exhausting to truly _know_ all the performance optimizations and [web performance acronyms][web-perf] that are helpful to make your site blazing fast. H/2, PRPL, RAIL, FLIP, SPA, SW, TTI, oh my! Gatsby internalizes many of these practices so that you get _for free_ a blazing fast site with smart defaults out of the box. It's valuable to show rather than tell, so let's go over some performance monitoring tools and techniques so that you can learn how to measure the speed of your site and measure your site's performance. In particular, I'll go over two tools: Lighthouse and WebPagetest, and two metrics, Time to Interactive (TTI) and Speed Index. Armed with the knowledge of these tangible metrics and tools you can empower yourself to _know_ whether your site is actually blazing fast — or merely an imposter.
+
+### Lighthouse / Google Audit
```

Comment:
We could link to some docs somewhere in this article. There are some docs on how to increase your lighthouse score in this category: https://www.gatsbyjs.org/docs/performance/

Would be good to get your feedback on those docs and their titles too! Maybe these performance docs ought to get a more prominent place in the docs in general, now that I think about it...